elytron/src/main/java/org/wildfly/extension/elytron/ModifiableRealmDecorator.java

```diff
@@ -303,7 +216,11 @@ protected void executeRuntimeStep(final OperationContext context, final ModelNod
 
     static class AttributeAddHandler extends ElytronRuntimeOnlyHandler {
 
-        public static final SimpleAttributeDefinition NAME = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.NAME, ModelType.STRING, false)
+        static final SimpleAttributeDefinition IDENTITY = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.IDENTITY, ModelType.STRING, false)
+                .setAllowExpression(false)
```

Comment:
Just an FYI -- calling this is unnecessary as it's the default. Calls like this increase boot time. Probably in undetectable amounts though. ;)